crm/types.go

```diff
@@ -58,4 +82,130 @@ type (
 		Color     string `json:"color"`
 		ID        int    `json:"id"`
 	}
+
+	// CRM EntityResponse, returns when entity object was created successfully
+	EntityResponse struct {
```

Comment:
It returns the same struct as we process, i.e. if we post `Company` to create, it will respond with created `Company`